src/main/webapp/app/exam/manage/exams/exam-update.component.ts

```diff
@@ -446,6 +450,69 @@ export class ExamUpdateComponent implements OnInit, OnDestroy {
             dayjs(this.exam.exampleSolutionPublicationDate).isBefore(this.exam.endDate || null)
         );
     }
+    /**
+     * Default exam start text, which can be edited by instructors in the text editor
+     */
+    get examDefaultStartText(): string {
+        const warningForInstructionsText =
+            '<!-- PLEASE MAKE SURE TO READ ALL INSTRUCTIONS, CHECK EXAM INFORMATION CAREFULLY, AND ADAPT THEM TO YOUR SPECIFIC USE CASES AS NEEDED -->\n\n';
```

Comment:
```suggestion
            '<!-- Please adapt this text for your exam -->\n\n';
```

Reply:
Yeah, I also think this would be more polite